### ISSUE TITLE
Cannot run task

### ISSUE THREAD

**Issue by huiyiqun** [ansible-semaphore/semaphore#117]
I have just installed a fresh Semaphore and added key, repo, inventory, task template. everything was OK.

But when I tried to run a task, the server exited.

```
panic: Error 1364: Field 'task' doesn't have a default value

goroutine 36 [running]:
panic(0xa5e780, 0xc8204c2ea0)
        /usr/local/Cellar/go/1.6/libexec/src/runtime/panic.go:464 +0x3e6
github.com/ansible-semaphore/semaphore/routes/tasks.(*task).log.func1(0xc8202cf080, 0xc8204b9dc1, 0xb, 0xeced89362, 0x22a2e24b, 0x102bd60)
        /Users/m/p/gospace/src/github.com/ansible-semaphore/semaphore/routes/tasks/logging.go:36 +0x26c
created by github.com/ansible-semaphore/semaphore/routes/tasks.(*task).log
        /Users/m/p/gospace/src/github.com/ansible-semaphore/semaphore/routes/tasks/logging.go:38 +0x640
2016/05/26 14:49:06 notifying bugsnag: Error 1364: Field 'task' doesn't have a default value
2016/05/26 14:49:06 bugsnag.Notify: not notifying in development
```

What's the matter?

**Comment by charlie-telecom** [comment]
I have this issue too. Moreover, the server shouldn't exit. This exception has to be caught.

**Comment by snevs** [comment]
Same issue here: 

GIN-debug] Listening and serving HTTP on :3000
[GIN] 2016/06/03 - 10:33:43 | 200 |      477.59µs | ::1 |   GET     /
[GIN] 2016/06/03 - 10:33:43 | 200 |   29.712581ms | ::1 |   GET     /public/css/semaphore.css
[GIN] 2016/06/03 - 10:33:43 | 200 |   57.737519ms | ::1 |   GET     /public/js/bundle.js
[GIN] 2016/06/03 - 10:33:44 | 200 |    3.844077ms | ::1 |   GET     /public/vendor/fontawesome/fonts/fontawesome-webfont.woff2
[GIN] 2016/06/03 - 10:33:44 | 200 |   49.376019ms | ::1 |   GET     /api/user
[GIN] 2016/06/03 - 10:33:44 | 200 |    21.21515ms | ::1 |   GET     /public/js/controllers/dashboard.js
[GIN] 2016/06/03 - 10:33:44 | 200 |   30.974916ms | ::1 |   GET     /api/info
[GIN] 2016/06/03 - 10:33:44 | 200 |     343.121µs | ::1 |   GET     /public/js/controllers/projects/edit.js
[GIN] 2016/06/03 - 10:33:45 | 200 |     101.723µs | ::1 |   GET     /public/html/dashboard.html
[GIN] 2016/06/03 - 10:33:45 | 200 |   10.525713ms | ::1 |   GET     /api/projects
[GIN] 2016/06/03 - 10:33:45 | 200 |   32.433099ms | ::1 |   GET     /api/events
[GIN] 2016/06/03 - 10:33:47 | 200 |     123.923µs | ::1 |   GET     /public/js/controllers/projects/dashboard.js
[GIN] 2016/06/03 - 10:33:47 | 200 |   10.762837ms | ::1 |   GET     /api/project/1
[GIN] 2016/06/03 - 10:33:47 | 200 |     198.689µs | ::1 |   GET     /public/html/projects/container.html
[GIN] 2016/06/03 - 10:33:47 | 200 |      75.571µs | ::1 |   GET     /public/js/controllers/projects/taskRunner.js
[GIN] 2016/06/03 - 10:33:47 | 200 |     292.781µs | ::1 |   GET     /public/html/projects/dashboard.html
[GIN] 2016/06/03 - 10:33:47 | 200 |    63.50761ms | ::1 |   GET     /api/project/1/events
[GIN] 2016/06/03 - 10:33:47 | 200 |  277.425838ms | ::1 |   GET     /api/project/1/tasks
[GIN] 2016/06/03 - 10:33:51 | 200 |     217.583µs | ::1 |   GET     /public/html/projects/taskModal.html
[GIN] 2016/06/03 - 10:33:51 | 200 |   43.666116ms | ::1 |   GET     /api/project/1/tasks/1/output
[GIN] 2016/06/03 - 10:33:53 | 200 |   11.880922ms | ::1 |   GET     /api/project/1/tasks/1/output
[GIN] 2016/06/03 - 10:34:01 | 200 |     513.358µs | ::1 |   GET     /public/js/controllers/projects/templates.js
[GIN] 2016/06/03 - 10:34:01 | 200 |     112.816µs | ::1 |   GET     /public/html/projects/templates/list.html
[GIN] 2016/06/03 - 10:34:02 | 200 |   37.796938ms | ::1 |   GET     /api/project/1/keys
[GIN] 2016/06/03 - 10:34:02 | 200 |   42.586388ms | ::1 |   GET     /api/project/1/templates
[GIN] 2016/06/03 - 10:34:02 | 200 |    43.22924ms | ::1 |   GET     /api/project/1/environment
[GIN] 2016/06/03 - 10:34:02 | 200 |   43.518973ms | ::1 |   GET     /api/project/1/inventory
[GIN] 2016/06/03 - 10:34:02 | 200 |   44.188914ms | ::1 |   GET     /api/project/1/repositories
[GIN] 2016/06/03 - 10:34:03 | 200 |     239.097µs | ::1 |   GET     /public/html/projects/createTaskModal.html
[GIN] 2016/06/03 - 10:34:14 | 201 |   80.901327ms | ::1 |   POST    /api/project/1/tasks
&{{2 4 waiting true oasg-update.yml  {63600536054 91683818 0x102bd60} <nil> <nil>} {0 0 0 0 0 <nil>  <nil> false} {0   <nil> <nil> <nil>} {0  0  <nil> {0   <nil> <nil> <nil>} <nil> {0   <nil> <nil> <nil>} } {0  0  0 {0   <nil> <nil> <nil>}} {0  0 <nil> } [] 1}
[1]
panic: Error 1364: Field 'task' doesn't have a default value

goroutine 38 [running]:
panic(0xa5e780, 0xc820474b00)
	/usr/local/Cellar/go/1.6/libexec/src/runtime/panic.go:464 +0x3e6
github.com/ansible-semaphore/semaphore/routes/tasks.(*task).log.func1(0xc820451340, 0xc8203fb6e1, 0xb, 0xecee329f6, 0xc02fc7d, 0x102bd60)
	/Users/m/p/gospace/src/github.com/ansible-semaphore/semaphore/routes/tasks/logging.go:36 +0x26c
created by github.com/ansible-semaphore/semaphore/routes/tasks.(*task).log
	/Users/m/p/gospace/src/github.com/ansible-semaphore/semaphore/routes/tasks/logging.go:38 +0x640
2016/06/03 10:34:14 notifying bugsnag: Error 1364: Field 'task' doesn't have a default value
2016/06/03 10:34:14 bugsnag.Notify: not notifying in development

**Comment by VanWade** [comment]
I have the same issue and do not know how to fix.

**Comment by sudorepo** [comment]
me too, and the server exists. what to do?

`/Users/m/p/gospace/src/github.com/ansible-semaphore/semaphore/routes/tasks/logging.go:38 +0x640
2016/06/09 11:44:39 notifying bugsnag: Error 1364: Field 'task' doesn't have a default value
2016/06/09 11:44:39 bugsnag.Notify: not notifying in development`

**Comment by metal3d** [comment]
So do I. Same error, I really don't understand...

**Comment by metal3d** [comment]
@matejkramny https://github.com/ansible-semaphore/semaphore/commit/f16b3a57e8412c95962f8b1f9a9f13d5398d7ce0 breaks here, I had to change table and set "task" default value to "" and not NULL.

The error is explicit: `Error 1364: Field 'task' doesn't have a default value`

Thanks !

**Comment by metal3d** [comment]
For other( @sudorepo, @snevs...) : connect to mysql, use your "semaphore" db and do this command:
```sql
use semaphore; 
alter table task__output change  task task varchar(255) default '';
```
That should work afterward.

**Comment by matejkramny** [comment]
The `task` field is there to support parsing of the output. Is not completed however.

This is fixed in f16b3a57e8412c95962f8b1f9a9f13d5398d7ce0 but unreleased so far.
